deployer/commands/generate/cryptnono_config/__init__.py

```diff
@@ -0,0 +1,34 @@
+import shutil
+import subprocess
+from pathlib import Path
+
+from deployer.cli_app import generate_app
+
+HERE = Path(__file__).parent
+REPO_ROOT_PATH = HERE.parent.parent.parent.parent
```

Comment:
I think we should import the one in utils so that if we ever want to restructure the deployer again or relocate this file, we don't need to update it everywhere, just in one place.

```suggestion
from deployer.cli_app import generate_app
from deployer.utils.file_acquisition import REPO_ROOT_PATH

HERE = Path(__file__).parent
```